patches/NDK_3.9/Include/include_h/graphics/gfxmacros.h.diff

```diff
@@ -0,0 +1,22 @@
+--- NDK_3.9/Include/include_h/graphics/gfxmacros.h.orig	2017-01-09 14:37:17.040582249 +0100
++++ NDK_3.9/Include/include_h/graphics/gfxmacros.h	2017-01-09 14:37:32.084233042 +0100
+@@ -55,9 +55,6 @@
+ #define SafeSetOutlinePen(w,c)	  {if (GfxBase->LibNode.lib_Version<39) { (w)->AOlPen = c;(w)->Flags |= AREAOUTLINE;} else SetOutlinePen(w,c); }
+ #define SafeSetWriteMask(w,m)	{if (GfxBase->LibNode.lib_Version<39) { (w)->Mask = (m);} else SetWriteMask(w,m); }
+ 
+-/* synonym for GetOPen for consistency with SetOutlinePen */
+-#define GetOutlinePen(rp) GetOPen(rp)
```

Comment:
It's clear that one should be careful in changing system headers. However, as you know I am maintaining quite an amount of big amiga software titles, thus I am quite confident that my proposed change won't break anything.

Regarding your proposed change (only deleting the second definition and swapping GetOpen and GetOutlinePen) I am quite unsure if the latter would be a good idea. How come that you believe swapping GetOutlinePen and GetOPen would be required?!?